web/Views/Property/PropertyInputForm/PropertyAccepts.cshtml

```diff
@@ -45,7 +45,7 @@
     <div class="form-check">
         <input type="checkbox" asp-for="AcceptsBenefits" class="form-check-input"/>
         <label asp-for="AcceptsBenefits" class="form-check-label">
-            on Universal Credit or housing benefits?
+            on Housing Benefits?

```

Comment:
Could you do me a favour and add a question mark to the one above (`on Universal Credit?`)